modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -40,22 +34,21 @@ Here is an example of a Mule configuration that utilizes this to expose an HTTPS
 </http:listener-config>
 ----
 
-You may also refer to this knowledge-base article: https://support.mulesoft.com/s/article/Sample-App-Configuring-HTTPS-endpoint-for-deployment-in-Cloudhub[Sample App: Configuring HTTPS endpoint for deployment in CloudHub]
-
-*Important*: On the Mule worker, the CloudHub load balancer proxies port :80 to :8081 for HTTP and proxies port :443 to :8082 for HTTPS. +
-The `http.port` property is automatically set to port 8081 for HTTP, and `https.port` is set to port 8082 for HTTPS. If other values for http.port and https.port are specified in the mule-app.properties file, these are overwritten at deployment time.
+[IMPORTANT]
+On the Mule worker, the CloudHub load balancer proxies port :80 to :8081 for HTTP and proxies port :443 to :8082 for HTTPS. +
```

Comment:
I'd write the number "port 80-8081 for HTTP and ...port 443-8082" for ultimate clarity...

Reply:
the colon isn't really part of the port number (just required when you code it), I think.